pkg/kvstore/export.go

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package kvstore
+
+import (
+	"context"
+
+	"github.com/cilium/cilium/pkg/inctimer"
+	client "go.etcd.io/etcd/client/v3"
+)
+
+var (
+	// etcdDummyAddress can be overwritten from test invokers using ldflags
+	etcdDummyAddress = "http://127.0.0.1:4002"
+)
+
+func EtcdDummyAddress() string {
+	return etcdDummyAddress
+}
+
+func (e *EtcdModule) setConfigDummy() {
+	e.config = &client.Config{}
+	e.config.Endpoints = []string{etcdDummyAddress}
+}
+
+func ExportedGetBackend(name string) backendModule {
```

Comment:
The fact that a given variable/method is exported is already implied by the name starting with an uppercase latter. No need to explicitly name it `Exported`. I do also see limited value in creating an alias name of an already existing function, it should be just exported if it needs to be so.